prow/entrypoint/run.go

```diff
@@ -150,6 +165,13 @@ func (o Options) ExecuteProcess() (int, error) {
 	cancelled, aborted := false, false
 	done := make(chan error)
 	go func() {
+		for scanner.Scan() {
+			if o.DateTimeFormat != "" {
```

Comment:
nit: We could avoid using the scanner entirely when `o.DateTimeFormat == ""`. I don't think this adds much overhead though so it probably doesn't matter.

Reply:
Let me run some benchmarks to gauge the performance difference between pipes/scanning/logging `output` and just setting stdout and stderr to `output. If it is considerable, I will branch the logic here.